### PR TITLE
#1600 - Resolve issues with missing ./configure messages

### PR DESCRIPTION
**Pull request by DuaneEllis-TI**
Issue #1600

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -353,6 +353,7 @@ AM_CONDITIONAL([OPENTHREAD_BUILD_TESTS], [test "${nl_cv_build_tests}" = "yes"])
 #  Full Thread Device
 #
 
+AC_MSG_CHECKING([Enable FTD])
```
Comment (jwhui) [inline configure.ac]: nit: looking at the other instances of `AC_MSG_CHECKING`, the wording seems to be more "colloquial".  For example:

```
checking for strip... strip
checking whether ln -s works... yes
checking for cmp... /usr/bin/cmp
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking whether gcc is Clang... yes
```

What about changing these message to be:

```
AC_MSG_CHECKING([whether to enable FTD])
```

**Comment by jwhui** [comment]
Thanks for addressing this issue!  See my comment above.

**Comment by DuaneEllis-TI** [comment]
Done.

File: configure.ac [hunk 2]
```diff
@@ -447,6 +451,7 @@ if test "$enable_builtin_mbedtls" = "yes" -a ! "${MBEDTLS_CPPFLAGS}"; then
     MBEDTLS_CPPFLAGS="${MBEDTLS_CPPFLAGS} -I\${abs_top_srcdir}/third_party/mbedtls/repo/include/mbedtls"
     MBEDTLS_CPPFLAGS="${MBEDTLS_CPPFLAGS} -DMBEDTLS_CONFIG_FILE=\\\"mbedtls-config.h\\\""
 fi
+AC_MSG_CHECKING([whether MBED DTLS should be enabled])
```
Comment (jwhui) [inline configure.ac]: nit: `MBED DTLS` -> `mbed TLS`

**Pull request merged**